moto/route53/responses.py

```diff
@@ -526,17 +546,26 @@ def reusable_delegation_set(self, request, full_url, headers):
         {% endif %}
         <PrivateZone>{{ 'true' if zone.private_zone else 'false' }}</PrivateZone>
       </Config>
+      {% if zone.private_zone %}
+      <VPC>
+        <VPCId>{{zone.vpcid}}</VPCId>
+        <VPCRegion>{{zone.vpcregion}}</VPCRegion>
+      </VPC>
+      {% endif %}
    </HostedZone>
+   {% if not zone.private_zone %}
    <DelegationSet>
       <Id>{{ zone.delegation_set.id }}</Id>
       <NameServers>
          {% for name in zone.delegation_set.name_servers %}<NameServer>{{ name }}</NameServer>{% endfor %}
       </NameServers>
    </DelegationSet>
-   <VPC>
```

Comment:
As far as I can tell, the VPC-config was in the correct location here. The docs say it's supposed to be at the top-level, not inside the `<HostedZone>`.
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/route53.html#Route53.Client.create_hosted_zone

Changing that might fix a few unit tests, but it would be good to get the rest passing as well.

Reply:
Yep, my bad. Thanks for pointing out.